docs/assets/javascripts/version-select.js

```diff
@@ -1,3 +1,5 @@
+// TODO delete the following line
+var testData =[{"version": "v10.0", "title": "Self-hosted v10.0", "aliases": []}, {"version": "v9.0", "title": "Self-hosted v9.0", "aliases": []}, {"version": "v8.1", "title": "Self-hosted v8.1", "aliases": []}, {"version": "v8.0", "title": "Self-hosted v8.0", "aliases": []}, {"version": "v7.0", "title": "Self-hosted v7.0", "aliases": []}, {"version": "v6.0", "title": "Self-hosted v6.0", "aliases": []}, {"version": "v5.1", "title": "Self-hosted v5.1", "aliases": []}, {"version": "v5.0", "title": "Self-hosted v5.0", "aliases": []}, {"version": "v4.4", "title": "Self-hosted v4.4", "aliases": []}, {"version": "v4.3", "title": "Self-hosted v4.3", "aliases": []}, {"version": "v4.2", "title": "Self-hosted v4.2", "aliases": []}, {"version": "v4.1", "title": "Self-hosted v4.1", "aliases": []}, {"version": "v4.0", "title": "Self-hosted v4.0", "aliases": []}, {"version": "v3.5", "title": "Self-hosted v3.5", "aliases": []}, {"version": "v3.4", "title": "Self-hosted v3.4", "aliases": []}, {"version": "v3.3", "title": "Self-hosted v3.3", "aliases": []}, {"version": "v3.2", "title": "Self-hosted v3.2", "aliases": []}, {"version": "v3.1", "title": "Self-hosted v3.1", "aliases": []}, {"version": "v3.0", "title": "Self-hosted v3.0", "aliases": []}, {"version": "v2.2", "title": "Self-hosted v2.2", "aliases": []}, {"version": "v2.1", "title": "Self-hosted v2.1", "aliases": []}, {"version": "v2.0.387-deprecated", "title": "Self-hosted v2.0.387 (Deprecated)", "aliases": []}, {"version": "v2.0", "title": "Self-hosted v2.0", "aliases": []}, {"version": "v1.5", "title": "Self-hosted v1.5", "aliases": []}, {"version": "v1.4", "title": "Self-hosted v1.4", "aliases": []}, {"version": "v1.3", "title": "Self-hosted v1.3", "aliases": []}, {"version": "v1.2", "title": "Self-hosted v1.2", "aliases": []}, {"version": "v1.1", "title": "Self-hosted v1.1", "aliases": []}, {"version": "v1.0", "title": "Self-hosted v1.0", "aliases": []}, {"version": "latest", "title": "Cloud (Latest)", "aliases": []}]
```

Comment:
```suggestion
```

we could clean this right?

Reply:
Correct. This data is here just because it is not available on the preview. I hard-coded it to test locally and to make it available on this preview.

The reason: If you check another preview deployment related to some other PR, say [the CS2 one](https://feat-multiple-coding-standards-io-3--docs-codacy.netlify.app/), you'll see that the dropdown only has 1 version, which wouldn't allow us to confirm visually that the sorting is working properly.